unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -576,9 +576,11 @@ static ERROR_MAP = {
         });
       }
       await this.delay(500);
+      const [baseUrl, queryString] = this.redirectUrl.split('?');
+      const additionalParams = unityConfig.env === 'stage' ? `${new URLSearchParams(window.location.search).toString()}&` : '';
```

Comment:
why do you need the new URLSearchParams ctor + toString here?  Does this achieve anything?

Reply:
Alternatively, you could construct a URLSearchParams() object, conditionally append a feedback prop, then have a single place where you set `window.location.href`